apps/ledger-live-mobile/src/components/DeviceAction/index.tsx

```diff
@@ -299,19 +387,18 @@ export function DeviceActionDefaultRendering<R, H, P>({
     return renderInWrongAppForAccount({
       t,
       onRetry,
-      accountName: inWrongDeviceForAccount.accountName,
       colors,
       theme,
     });
   }
 
-  if (imageLoadRequested) {
+  if (imageLoadRequested && device) {
     return renderImageLoadRequested({ t, device });
   }
-  if (loadingImage) {
+  if (loadingImage && device && progress) {
```

Comment:
progress can be `0` and it would be falsy here

Reply:
good catch 👍